.github/workflows/bot-crowdin-sync.yml

```diff
@@ -1,6 +1,10 @@
 name: "[Bot] Crowdin translations update"
 
-on: [workflow_dispatch]
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
```

Comment:
this is only for testing right? Or would you like to keep it like this?

Reply:
Yeah just testing, it's in a separate commit